src/styles/sectionPickerOverrides.less

```diff
@@ -27,6 +26,9 @@
 	border: 1px white solid;
 	font-family: @SegoeUi;
 	line-height: 30px;
+	width: 97%;
```

Comment:
I'm a little weary of this width declaration (it might cause issues on certain screen dpis, and is not resilient to change). What's the value of the width in the browser inspector without this declaration? I'm wondering if we can improve this.